api/api_sources/sources/database/database-schema/schema-sqls/SurveyGeometryCodeSchema.sql

```diff
@@ -0,0 +1,39 @@
+-- ### Creating Table: survey_geometry_code ### --
+
+        
+CREATE TABLE survey_geometry_code ();
+ALTER TABLE survey_geometry_code ADD COLUMN survey_geometry_code_id SERIAL PRIMARY KEY;
+ALTER TABLE survey_geometry_code ADD COLUMN survey_geometry_code VARCHAR(2) NOT NULL UNIQUE;
+ALTER TABLE survey_geometry_code ADD COLUMN description VARCHAR(100) NULL;
+ALTER TABLE survey_geometry_code ADD COLUMN active_ind BOOLEAN NOT NULL DEFAULT TRUE;
+
+
+        
+-- ### Creating Comments on table ### --
+
+        
+COMMENT ON TABLE survey_geometry_code IS 'Code table for schema of survey geometry codes';
```

Comment:
I'd like to recommend we add a more descriptive table comment that describes "survey geometry code" from a business perspective.

For example; Describes a 2 or 3 dimensional geometry used to describe the spatial extent of an observation: Point, linear corridor, transect, circle, rectangular plot, stratified grid, radio tract, GPS tract, water body outline.